src/main/java/com/netgrif/application/engine/pdf/generator/service/PdfDrawer.java

```diff
@@ -239,6 +246,14 @@ public void drawStroke(int x, int y, int fieldPosY, int width, int lineCounter,
         contentStream.stroke();
     }
 
+    @Override
+    public void drawLine(int x, int y, int fieldPosY, int width, int lineCounter, float strokeWidth) throws IOException {
+        contentStream.setStrokingColor(Color.LIGHT_GRAY);
```

Comment:
Why not make the colour of the stroke a method argument?

Reply:
Done